src/Data/Tree/AVL/Indexed/Relation/Unary/Any.agda

```diff
@@ -0,0 +1,118 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- AVL trees where at least one element satisfies a given property
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Relation.Binary using (StrictTotalOrder)
+
+module Data.Tree.AVL.Indexed.Relation.Unary.Any
+  {a ℓ₁ ℓ₂} (strictTotalOrder : StrictTotalOrder a ℓ₁ ℓ₂)
+  where
+
+open import Data.Nat.Base using (ℕ)
+open import Data.Product using (_,_; ∃; -,_; proj₁; proj₂)
+open import Data.Sum.Base using (_⊎_; inj₁; inj₂; [_,_]′)
+open import Function.Base
+open import Level using (Level; _⊔_)
+
+open import Relation.Nullary using (Dec; no)
+open import Relation.Nullary.Decidable using (map′)
+open import Relation.Nullary.Sum using (_⊎-dec_)
+open import Relation.Unary
+
+open StrictTotalOrder strictTotalOrder renaming (Carrier to Key)
+open import Data.Tree.AVL.Indexed strictTotalOrder as AVL
+  using (Tree; Value; Key⁺; [_]; _<⁺_; K&_; _,_; key; _∼_⊔_; ∼0; leaf; node)
+
+
+private
+  variable
+    v p q : Level
+    V : Value v
+    P : Pred (K& V) p
+    Q : Pred (K& V) q
+    l u : Key⁺
+    n : ℕ
+    t : Tree V l u n
+
+------------------------------------------------------------------------
+-- Definition
+
+-- Given a predicate P, Any P t is a path to one element in t that satisfies P.
```

Comment:
Very minor, but we should at least keep the comments under the character limit :smile: